README.md

```diff
@@ -37,7 +37,7 @@ npm install
 
 ### Testing changes
 
-Before testing Hyperspace, make the following change in `config.json`, located in the public directory:
+Before testing Hyperspace, make the following change in `config.json`, located in the `public` directory:
```

Comment:
Not sure if putting the formatting around `public` is completely necessary.

Reply:
The first time I read the README, I overlooked the fact that it was in a directory, since only `config.json` stood out and a lot of guide use the `directory/file` convention. I think that adding this formatting will make it harder to overlook this piece of information.